src/sentry/integrations/github/repository.py

```diff
@@ -142,4 +142,7 @@ def _transform_patchset(self, diff):
         return changes
 
     def pull_request_url(self, repo, pull_request):
-        return u'{}/pulls/{}'.format(repo.url, pull_request.key)
+        return u'{}/pull/{}'.format(repo.url, pull_request.key)
+
+    def repository_external_slug(self, repo):
+        return repo.name
```

Comment:
For Bitbucket and GitHub, is there a reason this isn't the `external_id`? What happens if the name changes?

Reply:
I saw your comment above! @markstory